### ISSUE TITLE
contrast issue on buttons with negative text-indent

### ISSUE THREAD

**Issue by WilcoFiers** [dequelabs/axe-core#1886]
This example throws a false positive, it shouldn't:

```html
<link href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet" />
<style>
	.form-submit {
		background: #d1c2a6;
	}
	.form-submit > input {
		height: 30px;
		background: transparent;
		margin-left: -18px;
		text-indent: -9999px;
		color: white;
		width: 30px;
	}
</style>
<span class="fa fa-search form-submit">
	<input type="submit" value="Search" />
</span>
```

Problem seems to have something to do with the combination of `input[type=button]` and `text-indent`. The problem goes away when using a `button` element.

**Comment by padmavemulapati** [comment]
Verified wit latest axe-coconut, didnot find any voilations on color-contrast for different input types and text-indent.

![image](https://user-images.githubusercontent.com/47409579/75784956-0096b180-5d89-11ea-88ad-311ce05e9d84.png)

**Comment by straker** [comment]
No docs needed
